test/unit/test_randomai.gd

```diff
@@ -1,7 +1,7 @@
 extends GutTest
```

Comment:
This currently just sets up a bunch of mirror matches, right? I wonder if it would be helpful to add a special test that runs two different random characters against each other; it might help to find a few more strange corner cases, if annotated properly

Reply:
Sure, we could have a test that just runs every character against every other character.
Probably take a while to run though, but it would have at least caught stuff like hazama vs critical